src/index.ts

```diff
@@ -268,6 +268,41 @@ export class API {
     });
   }
 
+  async getBots() {
+    return await this.request('GET', 1, '/bots');
+  }
+
+  async getBotsStats() {
+    return await this.request('GET', 1, '/bots/stats');
+  }
+
+  async getDeals(
+    limit: number = 50,
+    offset: number = 0,
+    scope: string = 'completed',
+    order:
+      | 'created_at'
+      | 'updated_at'
+      | 'closed_at'
+      | 'profit'
+      | 'profit_percentage' = 'created_at',
+    orderDirection: 'asc' | 'desc' = 'desc',
+    botId?: string
+  ) {
+    const params = {
+      limit,
+      offset,
+      scope,
+      order,
+      order_direction: orderDirection,
+    };
+
+    if (botId) {
+      params['bot_id'] = botId;
+    }
+    return await this.request('GET', 1, '/deals', params);
+  }
```

Comment:
Prefer allow user to pass in a `params` object and define the type in [here](https://github.com/kirosc/3commas-typescript/blob/master/src/types/types.ts)

So the function signature does not grow with the endpoint parameters

Reply:
Updated with proper typings, and the defaults passed in match 3commas documentation